### ISSUE TITLE
Ritualistic behavior

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6055]
**Preferred term label:**
Ritualistic behavior

**Synonyms**
Obsessive Behaviors

**Definition (free text, please give PubMed ID)**
Automatic behavior of psychogenic or cultural origin.

Source: https://medical-dictionary.thefreedictionary.com/ritualistic+behavior

**Parent term (use hpo.jax.org/app)**
Abnormal social behavior

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Obsessive Compulsive Behavior
Maybe ASD too?

**Comment**

Examples  include physical routines (needing to turn off the lights 10 times when leaving a room), needing to fix something that is not their version of correct (broccoli belongs on the right side of the plate and never on the left), staying on a strict schedule (showers only take place from 7-7:30pm), perseverating on a specific topic (talking only about dinosaurs), and needing specific questions answered by others (asking someone how old they are when you meet them). 

Source: https://www.behaviorfrontiers.com/blog/2019/8#:~:text=In%20the%20world%20of%20ABA,client%20and%20their%20entire%20family.

**Comment by pnrobinson** [comment]
Is the same as
https://hpo.jax.org/app/browse/term/HP:0000732
This term has no definition but is used to annotate autism-related diseases.
I do not think that obsessive behavior is an exact synonym.

**Comment by nicolevasilevsky** [comment]
oh yes, this term seems the same, we'll use this instead. Thanks Peter!
